_posts/2017-08-25-introducing-sandboxfs.md

```diff
@@ -3,40 +3,46 @@ layout: posts
 title: Introducing sandboxfs
 ---
 
+sandboxfs is a new sandboxing technique for Bazel. It's experimental and
```

Comment:
I added this tl;dr to make sure people don't think this is a new feature available now. Is this an accurate description? Please edit as you wish.

Reply:
Reworded a bit.